build/schema.js

```diff
@@ -13,21 +13,27 @@
             "properties": {
                 "alphabet": {
                     "type": "string",
-                    "minlength": 4,
-                    "maxlength": 4
+                    "pattern": "^[A-Z]{1}[a-z]{3}$"
                 },
                 "continent": {
-                    "type": "string",
-                    "minimum": 4,
-                    "maximum": 4
+                    "anyOf": [{
+                        "type": "string",
+                        "pattern": "^[A-Z]{2}$"
+                    }, {
+                        "type": "array",
+                        "items": {
+                            "type": "string",
+                            "pattern": "^[A-Z]{2}$"
+                        }
+                    }]
                 },
                 "language": {
                     "type": "string",
-                    "minlength": 1
+                    "pattern": "^[A-Za-z]+$"
```

Comment:
Good catch. I've forgot language variants in the API too.

For the language variants it may be helpful to have another property that indicates what you've said, e.g. `Mexico` in case of `Spanish`. What do you think? And if you agree, what would be a correct property name?